torch/csrc/autograd/functions/init.cpp

```diff
@@ -172,51 +172,51 @@ static struct PyGetSetDef batch_norm_backward_backward_properties[] = {
 static struct PyGetSetDef conv_forward_properties[] = {
   THP_FUNCTION_DEFAULT_PROPERTIES,
   {(char*)"stride", (getter)getTupleAttr<ConvForward, std::vector<int>, ConvParams,
-                                         &ConvParams::stride, long, PyInt_FromLong>, NULL, NULL, NULL},
+                                         &ConvParams::stride, int64_t, PyInt_FromLong>, NULL, NULL, NULL},
```

Comment:
The build fails on Linux/Mac with Python other than 2.7.9 here. It raises error like this from line 172 to 218:
```
error: insufficient contextual information to determine type
```